### ISSUE TITLE
Cannot pair phonegap mobile app with phonegap desktop app

### ISSUE THREAD

**Issue by wam19** [phonegap/phonegap-app-desktop#260]
I haven't been able to connect the phonegap app on my android with my phonegap project when I serve (either through the cli or the desktop app)

All the app does is say "connecting" and then "Timed Out". My phone and computer are on the same network, I've even tried to port forward the port that phonegap serve uses to make sure it wasn't being blocked.

It didn't work with Phonegap CLI version 3.5, so I updated it to 4.1, and it still doesn't work. (I have the most recent version of the phonegap app).

**Comment by hermwong** [comment]
Another issue with connecting could be related to AV software or firewall; if the port number that you are using with the PhoneGap Desktop App & PhoneGap Developer App is blocked by your AV software or firewall that would also block the pairing between PhoneGap Desktop & PhoneGap Developer.

By default the PhoneGap Desktop uses `port 3000`, however you can change the port in the PhoneGap Desktop's settings.

**Comment by hermwong** [comment]
See related: https://github.com/phonegap/phonegap-app-desktop/issues/162

**Comment by deepakgahlot98** [comment]
Hi,

As soon as i try to connect my phone it gives me an "download error"

my computer and my phone is connected to the guest wifi of my company 

i have tried to connect it at my home thorugh my home network as well, still the same message

the version of the application on the desktop is v0.1.8

**Comment by hermwong** [comment]
@deepakgahlot98, which version of the PhoneGap Developer App are you using on your phone?

**Comment by deepakgahlot98** [comment]
its 1.5.4

**Comment by hermwong** [comment]
Do you have firewall on your computer or on your router? Have you tried changing to another port number?

**Comment by deepakgahlot98** [comment]
no there is no firewall, i tried changing the port from 6000 to 3000 but that did not do any good :(

**Comment by eeijcea** [comment]
I have same issue ... I cannot access from Mobile Phone or Tablet .... However I can from browser in the same Laptop.. I am trying to debug sample onDeviceready App.. which from some reason seems not been triggered


![phonegap desktop error](https://cloud.githubusercontent.com/assets/3258578/10651255/c76f7ff0-784e-11e5-9455-c1e3e9d90583.png)

**Comment by eeijcea** [comment]
As you can see in the pic.... cordova.js file got a 500 error code ... which is the one I need to understand but for that I need debug from Phone capabilities..I do not think I can from Laptop

**Comment by hermwong** [comment]
You will not be able to see the deviceReady event being fired from your laptop/desktop; this event only gets fired on mobile devices (phones/tablets).

Sounds like you guys are running into #396, we are still investigating the issue.

I'll be locking this thread so that we can keep all discussion regarding the issue in the comments for #396.
